test/test_recognize.py

```diff
@@ -61,3 +62,16 @@ def test_recognize(workspace):
     assert os.path.exists(page1)
     with open(page1, 'r', encoding='utf-8') as f:
         assert 'verſchuldeten' in f.read()
+
+
+def test_recognize_should_warn_if_given_rgb_image_and_single_channel_model(workspace, caplog):
+    caplog.set_level(logging.WARNING)
+    CalamariRecognize(
+        workspace,
+        input_file_grp="OCR-D-GT-SEG-LINE",
+        output_file_grp="OCR-D-OCR-CALAMARI-BROKEN",
+        parameter={'checkpoint': CHECKPOINT}
+    ).process()
+
+    interesting_log_messages = [t[2] for t in caplog.record_tuples if "Using raw image" in t[2]]
+    assert len(interesting_log_messages) > 10  # For every line!
```

Comment:
Good idea!

Reply:
@mikegerber Once you revisit this, have a look at whether the tf impl is vulnerable to ￼	https://github.com/advisories/GHSA-977j-xj7q-2jr9 h/t @cneud